### PR TITLE
remove published to draft role for validator

### PR DESCRIPTION
**Pull request by KevinArtus**
[OO-MISC] Remove "published to draft' role for validator on fixtures.

https://github.com/open-orchestra/open-orchestra-model-bundle/pull/548
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/95
.https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1566

### CODE REVIEW

**Comment by ngilain** [comment]
It's not a BC break as you don't impact the code. I rather suggest a [OO-MISC] tag

:+1:

**Comment by KevinArtus** [comment]
@itkg-ngilain @alavieille I've been changed workflow function with new fixtures. tests seems working well

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace OpenOrchestra\WorkflowFunctionBundle\DataFixtures\MongoDB;
+
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use OpenOrchestra\ModelInterface\DataFixtures\OrchestraProductionFixturesInterface;
+use OpenOrchestra\WorkflowFunctionModelBundle\Document\WorkflowFunction;
+use OpenOrchestra\ModelInterface\DataFixtures\OrchestraFunctionalFixturesInterface;
```
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php]: remove this use

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataFunctional.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
 /**
  * Class LoadWorkflowFunctionData
  */
-class LoadWorkflowFunctionData extends AbstractFixture implements OrderedFixtureInterface, OrchestraProductionFixturesInterface, OrchestraFunctionalFixturesInterface
+class LoadWorkflowFunctionDataFunctional extends AbstractFixture implements OrderedFixtureInterface, OrchestraFunctionalFixturesInterface
```
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataFunctional.php:15]: update php doc

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+use OpenOrchestra\ModelBundle\Document\TranslatedValue;
+
+/**
+ * Class LoadWorkflowFunctionData
+ */
+class LoadWorkflowFunctionDataProduction extends AbstractFixture implements OrderedFixtureInterface, OrchestraProductionFixturesInterface
```
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php:15]: update php doc

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    /**
+     * Get the order of this fixture
+     *
+     * @return integer
+     */
+    public function getOrder()
+    {
+        return 125;
+    }
+
```
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php]: remove this empty line

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowRightDataFunctional.php [hunk 5]
```diff
@@ -14,7 +14,7 @@
 /**
  * Class LoadWorkflowRightData
  */
-class LoadWorkflowRightData extends AbstractFixture implements OrderedFixtureInterface, OrchestraProductionFixturesInterface, OrchestraFunctionalFixturesInterface
+class LoadWorkflowRightDataFunctional extends AbstractFixture implements OrderedFixtureInterface,OrchestraFunctionalFixturesInterface
```
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowRightDataFunctional.php:16]: update php doc

Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowRightDataFunctional.php:16]: remove this namespace not used OrchestraProductionFixturesInterface

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowrightDataProduction.php [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+namespace OpenOrchestra\WorkflowFunctionModelBundle\DataFixtures\MongoDB;
+
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use OpenOrchestra\ModelInterface\DataFixtures\OrchestraProductionFixturesInterface;
+use OpenOrchestra\WorkflowFunction\Model\WorkflowRightInterface;
+use OpenOrchestra\WorkflowFunctionModelBundle\Document\Authorization;
+use OpenOrchestra\WorkflowFunctionModelBundle\Document\WorkflowRight;
+use OpenOrchestra\ModelInterface\DataFixtures\OrchestraFunctionalFixturesInterface;
```
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowrightDataProduction.php]: remove this use

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataFunctional.php [hunk 7]
```diff
@@ -25,17 +24,17 @@ public function load(ObjectManager $manager)
...
 
         $enName = $this->generateTranslatedValue('en', 'Contributor');
         $frName = $this->generateTranslatedValue('fr', 'Contributeur');
         $workflowFunctionContributor = new WorkflowFunction();
         $workflowFunctionContributor->addName($enName);
         $workflowFunctionContributor->addName($frName);
-        $workflowFunctionContributor->addRole($this->getReference('role-draft'));
-        $this->addReference('workflow_function-contributor', $workflowFunctionContributor);
+        $workflowFunctionContributor->addRole($this->getReference('role-functional-published-to-draft'));
+        $this->addReference('workflow_function_contributor_functional', $workflowFunctionContributor);
```
Comment (ngilain) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataFunctional.php]: I'm not a big fan of mixing _ and - in the ref, you should choose one model and put it everywhere (maybe -)

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowrightDataProduction.php [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    /**
+     * Get the order of this fixture
+     *
+     * @return integer
+     */
+    public function getOrder()
+    {
+        return 1010;
+    }
+}
```
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowrightDataProduction.php]: eof

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowRightDataFunctional.php [hunk 9]
```diff
@@ -46,4 +45,4 @@ public function getOrder()
     {
         return 1000;
     }
-}
+}
```
Comment (ngilain) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowRightDataFunctional.php]: NL@EOF

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+}
```
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php]: eof

Comment (ngilain) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php]: NL@EOF

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowRightDataFunctional.php (hunk 9 above)
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowRightDataFunctional.php]: eof

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowrightDataProduction.php (hunk 8 above)
Comment (ngilain) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowrightDataProduction.php]: NL@EOF

**Comment by KevinArtus** [comment]
@alavieille Done, can you check it one more time pls

**Comment by alavieille** [comment]
:+1:

**Pull request merged**